frontend/views/containers/settings/UserProfile.vue

```diff
@@ -115,43 +94,46 @@
 
 <script>
 import { validationMixin } from 'vuelidate'
-import { email } from 'vuelidate/lib/validators'
+import validationsDebouncedMixins from '@view-utils/validationsDebouncedMixins.js'
+import { required, email } from 'vuelidate/lib/validators'
 import { OPEN_MODAL } from '@utils/events.js'
 import { cloneDeep } from '@utils/giLodash.js'
 import { mapGetters } from 'vuex'
 import imageUpload from '@utils/imageUpload.js'
 import Avatar from '@components/Avatar.vue'
+import BannerScoped from '@components/BannerScoped.vue'
 import sbp from '~/shared/sbp.js'
 import L from '@view-utils/translations.js'
 
 export default {
   name: 'UserProfile',
-  mixins: [validationMixin],
+  mixins: [validationMixin, validationsDebouncedMixins],
   components: {
-    Avatar
+    Avatar,
+    BannerScoped
   },
   data () {
     // create a copy of the attributes to avoid any Vue.js reactivity weirdness
     // so that we do not directly modify the values in the store
     const attrsCopy = cloneDeep(this.$store.getters.ourUserIdentityContract.attributes || {})
     return {
       form: {
-        picture: attrsCopy.picture,
-        bio: attrsCopy.bio,
         displayName: attrsCopy.displayName,
+        bio: attrsCopy.bio,
         email: attrsCopy.email
-      },
-      ephemeral: {
-        errorMsg: null,
-        newPicture: false,
-        profileSaved: false
       }
     }
   },
   validations: {
     form: {
-      picture: {},
-      email: { email }
+      email: {
+        [L('An email is required.')]: required,
+        [L('Please enter a valid email.')]: email,
+        [L('This email is already being used.')]: value => {
+          // TODO - verify if e-mail exists
+          return true
```

Comment:
This will never be done, so this check can be removed

Reply:
cc @mmbotelho  she asked this same question a couple of weeks ago.